src/Queue.py

```diff
@@ -26,8 +27,19 @@ def getQueueLength() -> int:
     return currQueue.count(where(BallChaserKey.ID).exists())
 
 
-def isPlayerInQueue(player: Member) -> bool:
-    return currQueue.contains(where(BallChaserKey.ID) == player.id)
+def isPlayerInQueue(player: Union[Member, str]) -> bool:
+    if (isinstance(player, Member)):
+        return currQueue.contains(where(BallChaserKey.ID) == player.id)
+    elif (isinstance(player, str)):
+        member = DataFiles.currQueue.get(doc_id=int(player))
+        if (member):
+            return currQueue.contains(where(BallChaserKey.ID) == member["id"])
+        else:
+            return False
+
+
+def getPlayerFromQueue(player: str) -> Union[BallChaser, None]:
+    return DataFiles.currQueue.get(doc_id=int(player))
```

Comment:
Should not need DataFiles. Just use currQueue.

Reply:
Resolved.